### ISSUE TITLE
Missing whitespace on inside of arbitrary parenthesis not reported/fixed

### ISSUE THREAD

**Issue by jrfnl** [WordPress-Coding-Standards/WordPress-Coding-Standards#1194]
Noticed something else which can do with some love while reviewing `src/wp-admin/install-helper.php` in combination with #1193: a condition within parenthesis in a control structure does not get whitespace on the inside of the parenthesis when running the fixer.

```php
if (($extra != null) && ($row->Extra != $extra)) {}

// Is fixed like:
if ( ($extra != null) && ($row->Extra != $extra) ) {}

// While I would expect:
if ( ( $extra != null ) && ( $row->Extra != $extra ) ) {}
```

**Comment by jrfnl** [comment]
@pento I've just been looking through the handbook. This is not explicitly mentioned there, but it is implied through the code sample used in the [Ternary section](https://make.wordpress.org/core/handbook/best-practices/coding-standards/php/#ternary-operator). It would also be in line with the other space usage rules.

What do you think ? Should this be added to the handbook and to the `Core` ruleset ? Or should this (for now) go into `Extra` ?

**Comment by pento** [comment]
This is definitely how spaces are supposed to be used, I've added an explicit mention to [the handbook](https://make.wordpress.org/core/handbook/best-practices/coding-standards/php/#space-usage).

**Comment by jrfnl** [comment]
@pento Ok, thanks for confirming. Sniff upcoming.

Regarding the handbook change: I don't think this only applies to _nested parentheses in control structures_, but I'd say it should apply to all "arbitrary" parentheses, including in function calls and ternaries.

Take these examples from Core - IMHO these should all be fixed:
```php
$sx = $vert ? ($w - 1) : 0;

$image->flip( ($operation->axis & 1) != 0, ($operation->axis & 2) != 0 );

if ( $success && ('nothumb' == $target || 'all' == $target) ) {}

$directory = ('/' == $file[ strlen($file)-1 ]);
```

**Comment by pento** [comment]
I knew I was being too specific with my wording as I wrote it. 🙂

Rewrote it as a catchall.

**Comment by jrfnl** [comment]
New sniff has been pulled both upstream as well as here - see #1195
